packages/pilot/src/components/MetricChart/charts/DonutChart.js

```diff
@@ -0,0 +1,64 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import {
+  applySpec,
+  map,
+  prop,
+} from 'ramda'
+import {
+  Cell,
+  Pie,
+  PieChart,
+  ResponsiveContainer,
+  Tooltip,
+} from 'recharts'
+
+const transformData = map(applySpec({
+  name: prop('label'),
+  value: prop('value'),
+}))
+
+const MetricDonutChart = ({
+  data,
+  height,
+  styles: { innerRadius },
+}) => (
+  <ResponsiveContainer height={height}>
+    <PieChart>
+      <Pie
+        dataKey="value"
+        data={transformData(data)}
+        innerRadius={innerRadius}
+        paddingAngle={2}
+      >
+        {
+          data.map((entry, index) => (
+            <Cell
+              // eslint-disable-next-line react/no-array-index-key
+              key={`${entry.value}-${index}`}
```

Comment:
Sugestão ramdafucks, meio bosta mas evia o `eslint-disable-line` acima
```suggestion
  key={concat([entry.value, '-', index])}
```

Reply:
aqui acho interessante usar `value` e `color` pra compor a key ao inves de index, isso deve garantir que não vai ter choque de valores